modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -9624,6 +9624,7 @@ localize-field-name=Localize Field Name
 localized=Localized
 localized-language=Localized Language
 localized-lookup=Localized Lookup
+localized-object-fields-can-not-be-used-in-object-actions=Localized object fields can not be used in object actions
```

Comment:
This language key came from a mockup?
I think it would be better to change "can not" to "must no". What do you think?

Reply:
No, this does not come from a mockup.
Makes sense, I will update it